chainercv/links/__init__.py

```diff
@@ -1,5 +1,6 @@
 from chainercv.links.model.pixelwise_softmax_classifier import PixelwiseSoftmaxClassifier  # NOQA
 
+from chainercv.links.model.faster_rcnn.faster_rcnn_train_chain import FasterRCNNTrainChain  # NOQA
```

Comment:
This import is against to the rule that we only import models which have APIs of detection or semantic segmentation.
I know `PixelwiseSoftmaxClassifier` is against too. Perhaps, we have to change the rule. Do you have any ideas?